vllm/core/scheduler.py

```diff
@@ -313,18 +366,19 @@ def _schedule_decodes(
         running_queue = policy.sort_by_priority(now, running_queue)
 
         while running_queue:
-            # NOTE: running
             seq_group = running_queue[0]
-            num_running_tokens = (
-                seq_group.num_seqs(status=SequenceStatus.RUNNING) *
-                self.num_decoding_tokens_per_seq)
+            num_running_tokens = self._get_num_new_tokens(
+                seq_group, SequenceStatus.RUNNING, enable_chunking, budget)
+            # We can have up to 1 running prefill at any given time in running
+            # queue, which means we can guarantee chunk size is at least 1.
+            assert num_running_tokens != 0
```

Comment:
This can't happen now. (because for normal case, if prefill is scheduled, this is not called. In chunked prefill, decoding is called first. I don't think we need to handle num_max_batched_tokens=0). So I will leave it in a current way and not handle this case (we can remove assert if there's other cases that this can happen)